### ISSUE TITLE
Questions regarding Training(parameters)

### ISSUE THREAD

**Issue by OrkunYilmaz** [XingangPan/SCNN#74]
Hello what kind of loss/error is Err1 and Err2 and the value in the brackets during training process?

Also could you please provide short informations about the following training parameters:

-backWeight
-maxIter
-retrain
-nThreads
-nEpochs

Much thanks in advance

**Comment by XingangPan** [comment]
@OrkunYilmaz Err1 is semantic segmentation loss (spatial cross entropy), Err2 is classification loss (cross entropy)
-backWeight: loss ratio for the background
-maxIter: maximum number of iterations during training
-retrain: path of the pretrained model to load
-nThreads: number of thread to load images
-nEpochs: maximum number of epochs during training
-shareGradInput: Share gradInput tensors to reduce memory usage

**Comment by OrkunYilmaz** [comment]
Thankyou for your reply!
I have one last question.
I am getting only FPs for the crossroads. Do you know why?

file: ./output/out7_cross.txt
tp: 0 fp: 956 fn: 0
precision: 0
recall: -1
Fmeasure: 0
